test/helpers/device-helpers.js

```diff
@@ -1,24 +1,34 @@
 import UUID from 'uuid-js';
-import * as simctl from 'node-simctl';
 import { retryInterval } from 'asyncbox';
-
+import Simctl from 'node-simctl';
 
 const MODEL = process.env.DEVICE_NAME || 'iPhone 8';
 const PLATFORM_VERSION = process.env.PLATFORM_VERSION || '13.2';
 
-async function createDevice (opts = {}) {
+async function prepareDevice (opts = {}) {
   const {
     model = MODEL,
     platformVersion = PLATFORM_VERSION,
+    prebooted = true,
   } = opts;
-  return await simctl.createDevice(`appium-idb-tests-${UUID.create().hex.toUpperCase()}`,
+  const simctl = new Simctl();
+  simctl.udid = await simctl.createDevice(
```

Comment:
Why is this needed, and not done in the `createDevice` function?

Reply:
I wanted to make it more explicit for the case createDevice builds multiple devices in a loop

Reply:
Ah. ok.